cpp/ps0004/cpp_ps0004.cpp

```diff
@@ -41,17 +24,16 @@ int main()
             std::cout << "Invalid character/s or zero value!" << std::endl;
             return 0;
       }
-
+      
       // check on convertedValue after passing float numbers and other values except int--
       if(convertedValue < 32 || convertedValue > 126) {
             std::cout << "Invalid: number is out of the range!" << std::endl;
-            return 0;
+            return 3;
       }
 
-
-      // printing the number and the equivalent printable character-------
+      // printing the number and the equivalent printable character------- 2/2
       std::cout << "The character for " << convertedValue << " is " << static_cast<char>(convertedValue) << std::endl;
 
-
       return 0;
-}
\ No newline at end of file
+}
+// great work 😎 10/10 ⭐

```

Comment:
- [x] program is in the correct path
- [x] no syntax errors, bugs or warnings while compilation and execution
- [x] formatted code to be clear and readable
- [x] commented some phrases for blocks of code
- [ ] main returns non-zero/zero number according to the case
+0.5 bonus mark for your efforts ⭐

you can check the [problem solution](https://onlinegdb.com/9x6wLTlkK) for more